src/components/select/select.js

```diff
@@ -228,9 +228,10 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
         });
       }
 
-      if (formCtrl) {
+      if (formCtrl && angular.isDefined(attr.multiple)) {
```

Comment:
So if we are **not using multiple**, then we **never `$setPristine()`? Is that valid logic?